maistra/ci/post-submit.sh

```diff
@@ -12,6 +12,9 @@ DIR=$(cd "$(dirname "$0")" ; pwd -P)
 # shellcheck disable=SC1090
 source "${DIR}/common.sh"
 
+# modify include paths to avoid overly long gcc command line
+fix_include_paths()
```

Comment:
you can move this call (without `()`) to the common.sh file

Reply:
This and below: not sure what you mean -- paths should only be modified once per checkout, can't invoke the update from bazel_build/bazel_test...

Reply:
You are calling this function in both files. You can just move this call to the bottom common.sh, which is included in pre and post submit files.

Reply:
done